.github/workflows/_deploy-analytics-platform.yml

```
@@ -0,0 +1,87 @@
+      group: ${{ format('{0}-{1}', inputs.script_name, inputs.deploy_env) }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.deploy_env }}
+    steps:
+      - name: Log Environment
+        run: |
+          env
+          echo "ECR_REPO_URI: ${{ inputs.ECR_REPO_URI }}"
+          echo "AWS_REGION: ${{ inputs.AWS_REGION }}"
+        shell: bash
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)